docs/definitionOfDone.md

```diff
@@ -0,0 +1,11 @@
+# Definition of Done
```

Comment:
Should the Definition of Done not be what was agreed upon and listed in the Github Issue? There seems to be extra items here which we will have to discuss as team before we add them in

Reply:
If we strictly follow the GitFlow workflow, then there must be two main branches, (main, develop). All others are supporting branches, I referred to the link for the original proposer and also the article link I posted on slack. 

I have no objection to edit the files in question.